### DISCUSSION TITLE
Reference in documentation

### DISCUSSION THREAD

**Discussion by a-ws-m** [Becksteinlab/MDPOW#279] [Q&A] [answered]
Hi @orbeckst , I had a question about this section of the docs: https://mdpow.readthedocs.io/en/latest/fep.html#differences-to-published-protocols
Can you send the reference to the Mobley paper you mention, please?
Also, intuitively I suspect that for the sake of the partition coefficient, there should be no need to do the vacuum calculation because it would cancel out when one takes the difference in free energies. But I'm getting very spurious results so perhaps that's wrong, hence why I'd like to review the paper.

**Comment by orbeckst** [comment, marked as answer]
This looks like a very old piece of the docs as it mentions GROMACS 4.x... I am afraid I don't remember which paper I referred to (should have written it down at the time!) — probably one of the earliest SAMPL papers from @dmobley's group.

There are issues with `couple-intramol = no` related to cut-offs, see https://github.com/Becksteinlab/MDPOW/blob/120c5dddd67d14c493971125e6355414b09e73c1/mdpow/templates/bar_charmm.mdp#L47-L50 (but I haven't looked at what the latest state there is).
